.nvmrc

```diff
@@ -1 +1 @@
-v16.17.1
+v18.17.1
```

Comment:
Just as a heads up, we can't use any nodejs 18 specific features until VS Code itself runs our extension using NodeJS 18, which will probably land in the September update (which will probably be out mid September)

Reply:
Changing the Node.js version here doesn't automatically leads to build (or whatever else) to immediately start using the new Node.js version.  The version that is currently configured will still be used independently of what you have written in `.nvmrc` until you apply it by `nvm use`, won't it"